recipes/isl/build.sh

```diff
@@ -0,0 +1,4 @@
+./configure --prefix="$PREFIX" --with-gmp-prefix="$PREFIX"
```

Comment:
I have set `CXXFLAGS` and `LINKFLAGS` as it was done in vigra feedstock: https://travis-ci.org/conda-forge/staged-recipes/jobs/122739248

I have also tried to set `CC` to `clang` and `CXX` to `clang++`: https://travis-ci.org/conda-forge/staged-recipes/jobs/122745881

Both attempts failed on OS X. BTW, have you noticed that the issue is on `make check` step *after* the library itself is built successfully?